services/app-api/src/emailer/templates.test.ts

```diff
@@ -19,14 +19,20 @@ import {
 import { formatRateNameDate } from '../../../app-web/src/common-code/dateHelpers'
 import { unlockedWithFullContracts } from '../../../app-web/src/common-code/healthPlanFormDataMocks'
 
+const stateAnalystEmails = [
```

Comment:
For this kind of test data, I strongly recommend making this a function that you call that returns the array rather than just a const array b/c that way if any individual test ends up modifying the array it won’t affect any other tests b/c they will get a fresh array when they call the function.